test_workloads.py

```diff
@@ -287,7 +288,7 @@ def test_gsc_java_spring_boot(self):
         assert("Hello from Graminized Spring Boot Application." in gsc_java_springboot_log)
 
     @pytest.mark.examples
-    @pytest.mark.skipif((os_release_id in ['alpine'] or ((int(no_cores) < 16) and sgx_mode == '1')),
+    @pytest.mark.skipif((os_release_id in ['alpine'] or (base_os == "ubuntu24.04") or ((int(no_cores) < 16) and sgx_mode == '1')),
```

Comment:
Similar comment, to merge base_os for alpine and ubuntu 24.04

Reply:
Modified.